enhancements/installer/aws-lb-subnet-selection.md

```diff
@@ -0,0 +1,686 @@
+need a way to configure ingress subnets at install time (Day 1).
+
+Configuring subnets at install time is essential for cluster admins that want to
+ensure their default IngressController is configured properly from the start. Additionally,
+Service Delivery has use cases for ROSA Hosted Control Plane (HCP) that require specific
+IngressController subnets to be configured during cluster installation.
+
+### User Stories
+
+#### Day 1 Ingress Controller Subnet Selection on AWS
+
+_"As a cluster admin, when installing a cluster in AWS, I want to specify the
+default subnet selection for my IngressControllers."_
+
+See [Setting IngressController Subnets during Installation Workflow](#setting-ingresscontroller-subnets-during-installation-workflow)
+for the workflow for this user story.
+
+#### Installing a Private Cluster with Private Subnets in AWS
+
+_"As a cluster admin, when installing a private cluster in AWS, I want to
+specify that the default IngressController should use private subnets, so
+that it is not exposed to the public Internet."_
+
+When a cluster admin installs a private cluster into a VPC also containing
+public subnets lacking the proper tags, internal LoadBalancer-type services
+may inadvertently use both the public and private subnets, unknowingly introducing
+a security risk (see related [RFE-2816](https://issues.redhat.com/browse/RFE-2816) and
+[OCPBUGS-17432](https://issues.redhat.com/browse/OCPBUGS-17432)).
+
+As a workaround to this type of issue, cluster admins may want to specify the private
+subnet list to ensure that the default IngressController exclusively uses the private
+subnets. See [Setting IngressController Subnets during Installation Workflow](#setting-ingresscontroller-subnets-during-installation-workflow)
+for the workflow for this user story.
+
+#### ROSA HCP: 1 MachinePool in 1 Availability Zone
+
+_"As a user of ROSA HCP, I want to install a cluster with 1 MachinePool in 1
+Availability Zone and have my default IngressController only mapped to the subnets
+in that Availability Zone."_
+
+Similar to [Day 1 IngressController Subnet Selection on AWS](#day-1-ingress-controller-subnet-selection-on-aws).
+
+When a cluster is installed into a VPC containing other subnets in different
+AZs than the cluster subnets, the default IngressController may map to subnets
+in the AZs that are not part of the cluster. This creates a security risk
+because the load balancer is mapping to subnets that may belong to other clusters. This
+user story is derived from [OCM-5969](https://issues.redhat.com/browse/OCM-5969).
+
+See [Setting IngressController Subnets during Installation Workflow](#setting-ingresscontroller-subnets-during-installation-workflow)
+for the workflow for this user story.
+
+#### ROSA HCP: Adding a 2nd MachinePool
+
+_"As a user of ROSA HCP, I want to install a cluster with 1 MachinePool in 1
+AZ, then add a 2nd MachinePool in another AZ and update my IngressController's
+subnets to map to the new AZ."_
```

Comment:
This use case is not an installation option, is it?  If not, it doesn't belong in this proposal.